docs/source/cluster_submission.rst

```diff
@@ -61,7 +69,7 @@ The submission scripts are generated using the same templating system like the `
 
 .. tip::
 
-    Use the ``--pretend`` or ``--test`` option to pre-view the generated submission scripts on screen instead of submitting them.
+    Use the ``--pretend`` or ``--test`` option to pre-view the generated submission scripts on screen instead of submitting them. Use the ``--exec`` to submit **all eligible** job-operations anyway, ignoring the groups tags.
```

Comment:
What this does is more akin to running all condition checks and such on the front end before submit rather than after submitting. In general the same thing happens with minimal overhead even if `--exec` mode isn't specified and signac `run` commands are used.